MetaMorpheusSetup/MetaMorpheusSetup.wixproj

```diff
@@ -1,31 +1,22 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project>
+  <Import Project="Sdk.props" Sdk="WixToolset.Sdk" Version="4.0.0-rc.2" />
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">x64</Platform>
-    <ProductVersion>3.10</ProductVersion>
-    <ProjectGuid>be72541a-66a4-4958-9d19-56acc7d3876b</ProjectGuid>
-    <SchemaVersion>2.0</SchemaVersion>
     <OutputName>MetaMorpheusInstaller</OutputName>
-    <OutputType>Package</OutputType>
     <Name>MetaMorpheusSetup</Name>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
```

Comment:
FireGiant's convertion extension deleted this line, changing the msi output path and ultimately throwing an error on appveyor due not finding the msi.